web/src/engine/websites/Mangadon.ts

```diff
@@ -77,6 +91,10 @@ export default class extends DecoratableMangaScraper {
         return (data as APIItem).attributes.pages.map(page => new Page(this, chapter, new URL(page.path, this.imgCDN), { ...cookies }));
     }
 
+    async checkAuth(): Promise<void> {
```

Comment:
May remove method that consist of a single line and is only used once

Reply:
Fine. Avoiding "called once methods" is a given ofc. I did it like that just in case , and to match other more complex auth code we may use elsewhere. Guess i was overthinking :D